sql/mob_pools.sql

```diff
@@ -3261,7 +3261,7 @@ INSERT INTO `mob_pools` VALUES (3204,'Progenerator','Progenerator',138,0x00006D0
 INSERT INTO `mob_pools` VALUES (3205,'Promathia','Promathia',134,0x0000300400000000000000000000000000000000,7,7,3,240,100,0,0,0,0,18,6172,32,3979,155,0,0,127,0,0,134,134);
 INSERT INTO `mob_pools` VALUES (3206,'Propagator','Propagator',138,0x00006D0400000000000000000000000000000000,1,1,3,240,100,0,1,1,0,2,0,0,2629,135,13,0,0,0,0,138,138);
 INSERT INTO `mob_pools` VALUES (3207,'Proteus','Proteus',218,0x0000610100000000000000000000000000000000,2,2,7,360,100,0,1,0,0,2,0,32,236,157,0,0,0,0,0,218,218);
-INSERT INTO `mob_pools` VALUES (3208,'Proto-Omega','Proto-Omega',54,0x0000370400000000000000000000000000000000,2,1,6,240,100,1024,1,1,0,18,6228,32,2773,157,1,0,0,0,0,727,518);
+INSERT INTO `mob_pools` VALUES (3208,'Proto-Omega','Proto-Omega',54,0x0000370400000000000000000000000000000000,2,1,6,240,100,1024,1,1,0,18,6230,32,2773,157,1,0,0,0,0,727,518);
```

Comment:
I believe the preference going forward was to use `onMobInitialize` to set immunities and define them as zero in sql for NMs

Reply:
> I believe the preference going forward was to use `onMobInitialize` to set immunities and define them as zero in sql for NMs

Yeah, that is mainly what we do on ASB/Horizon but I was not sure about on LSB. I will modify the PR to use that notation.